submissions/strumerlz/html-css-popup/style.css

```diff
@@ -0,0 +1,225 @@
+:root {
+    --hover-focus-background: #d6d6d6;
+    --basic-color: #595959;
+    --border-color: transparent;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html{
+    font-family: 'Poppins', sans-serif;
+    color: var(--basic-color);
+       
```

Comment:
could you please format your file by any auto-formatting? (it removes empty lines, adds necessary alignments)

Reply:
Corrected it.